### ISSUE TITLE
UNWIND returning empty resultset

### ISSUE THREAD

**Issue by kurtcorbett** [orientechnologies/orientdb#5657]
##### my local machine is running 2.1.9 (upgraded from 2.1.6). The following query returns empty results (In Studio and when using orientJS).
```
SELECT
  id,
  out('posted_to').name as page
FROM post

UNWIND
  page
```
<img width="1409" alt="screen shot 2016-01-29 at 7 13 24 pm" src="https://cloud.githubusercontent.com/assets/7966496/12692973/d0ebd3a0-c6bc-11e5-81cb-28dfc5e15a8a.png">

##### When I remove the 'UNWIND', i get data back, but the name field is an empty array.  

<img width="1393" alt="screen shot 2016-01-29 at 7 14 37 pm" src="https://cloud.githubusercontent.com/assets/7966496/12692981/e7a92d7c-c6bc-11e5-986e-0f24fda3842c.png">

##### I have tried the same query on a remote server (running 2.1.7) and it returns data correctly.

<img width="1410" alt="screen shot 2016-01-29 at 7 12 53 pm" src="https://cloud.githubusercontent.com/assets/7966496/12692965/aaefeaa6-c6bc-11e5-96d1-71b5e8606a7f.png">

**Comment by luigidellaquila** [comment]
Hi @kurtcorbett 

yes, it seems to be a bug, I'm on it

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @kurtcorbett 

I fixed the problem in 2.1.x branch, it will be released with next hotfix

Thanks

Luigi
